fetch.bs

```diff
@@ -4482,6 +4498,13 @@ Range Requests</cite>. [[HTTP-RANGE]] However, this is not widely supported by b
  <li><p>If <i>authentication-fetch flag</i> is set, then create an <a>authentication entry</a>
  for <var>request</var> and the given realm.
 
+ <li><p>If <i>revalidatingFlag</i> and <var>request</var>'s <a for=request>allows stale content flag</a>
+ are set, then in parallel perform <a for=main>main fetch</a> using <var>request</var> with
```

Comment:
Why "main fetch" and not "HTTP fetch"?

Reply:
There are some CORS checks in main and it is consistent with the other parts of the fetch spec.